engine/src/main/java/com/datatorrent/stram/client/StramAgent.java

```diff
@@ -421,7 +421,11 @@ private StramWebServicesInfo retrieveWebServicesInfo(String appId)
         permissionsInfo = new JSONObject(IOUtils.toString(is));
       } catch (IOException ex) {
         // ignore if file is not found
-        LOG.info("Exception in accessing permissions.json", ex);
+        if (LOG.isDebugEnabled()) {
```

Comment:
The difference is in expectations and code readability. It is quite unexpected that code path execution results in the FileNotFoundException almost 100%. I would ask to add a comment in case an agreement is to use exception as a mechanism to check if the file exists or not.